spec/01-unit/03-prefix_handler_spec.lua

```diff
@@ -154,6 +154,14 @@ describe("NGINX conf compiler", function()
       assert.matches("worker_connections %d+;", nginx_conf)
       assert.matches("multi_accept on;", nginx_conf)
     end)
+    it("converts dns_resolver to string", function()
+      local nginx_conf = prefix_handler.compile_nginx_conf({
+        dns_resolver = { "8.8.8.8", "8.8.4.4" }
+      }, [[
+        "resolver ${{DNS_RESOLVER}} ipv6=off;"
```

Comment:
I mean that if user wants the Nginx resolver use the same resolvers as Kong DNS resolver and configure it in one place, does it hurt?